geom/type_multi_polygon.go

```diff
@@ -9,59 +9,71 @@ import (
 )
 
 // MultiPolygon is a planar surface geometry that consists of a collection of
-// Polygons. The zero value is the empty MultiPolygon (i.e. the collection of
-// zero Polygons). It is immutable after creation.
-//
-// For a MultiPolygon to be valid, the following assertions must hold:
-//
-// 1. It must be made up of zero or more valid Polygons (any of which may be empty).
-//
-// 2. The interiors of any two polygons must not intersect.
-//
-// 3. The boundaries of any two polygons may touch only at a finite number of points.
+// (possibly empty) Polygons. The zero value is the empty MultiPolygon (i.e.
+// the collection of zero Polygons). It is immutable after creation.
 type MultiPolygon struct {
 	// Invariant: ctype matches the coordinates type of each polygon.
 	polys []Polygon
 	ctype CoordinatesType
 }
 
-// NewMultiPolygon creates a MultiPolygon from its constituent Polygons. It
-// gives an error if any of the MultiPolygon assertions are not maintained. The
+// NewMultiPolygon creates a MultiPolygon from its constituent Polygons. The
 // coordinates type of the MultiPolygon is the lowest common coordinates type
```

Comment:
I'm having a bit of trouble parsing this sentence, especially the "type its Polygons" part: 

> The coordinates type of the MultiPolygon is the lowest common coordinates type its Polygons.

Reply:
Thanks, yep I was missing an "of". Fixed in 8425f76b10062fe1fe5a62ede9998952b435a387.